### PR TITLE
[#170] Exclude backup_label.old from the backup

### PR DESCRIPTION
**Pull request by Jubilee101**

### CODE REVIEW

File: src/libpgmoneta/wf_restore.c [hunk 1]
```diff
@@ -50,7 +50,6 @@ static int recovery_info_teardown(int, char*, struct node*, struct node**);
 static char* get_user_password(char* username);
 static void create_standby_signal(char* basedir);
 
-
```
Comment (jesperpedersen) [inline src/libpgmoneta/wf_restore.c:53]: Skip white space change - so revert

File: src/main.c [hunk 2]
```diff
@@ -1925,7 +1925,7 @@ init_replication_slots(void)
          if (create_slot && slot_status == SLOT_NOT_FOUND && strlen(config->servers[srv].wal_slot) > 0)
          {
             create_slot = config->servers[srv].create_slot == CREATE_SLOT_YES ||
-               (config->create_slot == CREATE_SLOT_YES && config->servers[srv].create_slot != CREATE_SLOT_NO);
+                          (config->create_slot == CREATE_SLOT_YES && config->servers[srv].create_slot != CREATE_SLOT_NO);
```
Comment (jesperpedersen) [inline src/main.c:1928]: Skip white space change - so revert

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

**Comment by Jubilee101** [comment]
I didn't do it, it was uncrustify. Did you update your version? I remember
we are using the same version last time.

File: src/libpgmoneta/wf_restore.c (hunk 1 above)
Comment (Jubilee101) [reply src/libpgmoneta/wf_restore.c:53]: I reverted the changes, and then uncrustify just changed it back again.  Which version of uncrustify are you using right now?

**Comment by jesperpedersen** [comment]
Just change it, and we can do a separate pull request with uncrustify changes - this will make easier to back-port

**Comment by jesperpedersen** [comment]
Rebased, and merged.

Thanks for your contribution !

**Pull request closed without merging**